# The state of a robot together with a time when this state occurs along a trajectory

RobotStateCartesian  state
float64              time